### PR TITLE
{tools}[foss/2020a] IRkernel v1.2 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+name = 'IRkernel'
+version = '1.1.1'
+versionsuffix = '-R-%(rver)s-Python-%(pyver)s'
+
+homepage = 'https://irkernel.github.io'
+description = """The R kernel for the 'Jupyter' environment executes R code
+ which the front-end (Jupyter Notebook or other front-ends) submits to the
+ kernel via the network."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```
Comment (Micket) [inline easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb:12]: So, just to check, you know there is a version for foss 2020a already?

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/',
+        'https://cran.rstudio.com/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s/',
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('repr', '1.1.0', {
```
Comment (Micket) [inline easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb:34]: Maybe there is a reason not to update, but there seem to be newer versions available.

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        'checksums': ['743fe018f9e3e54067a970bc38b6b8c0c0498b43f88d179ac4a959c2013a5f96'],
+    }),
+    ('IRdisplay', '0.7.0', {
```
Comment (Micket) [inline easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb:37]: again, IRdisplay 1.0 is available

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        'checksums': ['91eac9acdb92ed0fdc58e5da284aa4bb957ada5eef504fd89bec136747999089'],
+    }),
+    ('pbdZMQ', '0.3-3', {
```
Comment (Micket) [inline easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb:40]: and here as well version 0.3-5

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/i/IRkernel/IRkernel-1.1.1-foss-2020a-R-4.0.0-Python-3.8.2.eb:12]: You are right, I didn't consider there was 2 R's

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/cc6777ebac73fbe924b0b2c05ba11957 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3537.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a273009685f11dabee67cce70b840d17 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @arielzn!

**Pull request merged**